sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -51,6 +51,42 @@ import org.apache.spark.sql.types._
 import org.apache.spark.sql.util.{CaseInsensitiveStringMap, SchemaUtils}
 import org.apache.spark.util.ArrayImplicits._
 
+/**
+ * Interface providing util to convert JValue to String representation of catalog entities.
+ */
+trait MetadataHashMapTranslator {
+  def jsonToString(jsonMap: mutable.LinkedHashMap[String, JValue]):
+  mutable.LinkedHashMap[String, String] = {
```

Comment:
nit: let's follow the style guide https://github.com/databricks/scala-style-guide?tab=readme-ov-file#indent
```
def newAPIHadoopFile[K, V, F <: NewInputFormat[K, V]](
    path: String,
    fClass: Class[F],
    kClass: Class[K],
    vClass: Class[V],
    conf: Configuration = hadoopConfiguration)
  : RDD[(K, V)] = {
  // method body
}
```